runbotics/runbotics-scheduler/src/mail/mail.service.ts

```diff
@@ -53,18 +60,62 @@ export class MailService {
     }
 
     public async sendBotDisconnectionNotificationMail(bot: IBot, installationId: string) {
-        await this.sendMail({
-            to: bot.user.email ?? '',
-            subject: NOTIFICATION_MAIL_SUBJECT,
-            content: `Hello,\n\nBot 🤖 (${installationId}) has been disconnected.\n\nBest regards,\nRunBotics`,
-        });
+        const disconnectedBot = await this.botService.findById(bot.id);
+
+        if (disconnectedBot.subscribers && disconnectedBot.subscribers.length) {
```

Comment:
imo we should notify each person that has access to a collection in which the bot is registered. In case the collection is public then send to anyone subscribed